manifests/i/isaaclevin/presencelight/5.4.11/isaaclevin.presencelight.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 5.4.11

```

```suggestion
PackageVersion: 5.4.11.0
```
to match Control Panel.